lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -81,12 +95,29 @@ def active_step_by_step
         @active_step_navs.first
       end
 
+      def secondary_step_by_step?
+        secondary_step_by_steps.any? && step_navs_combined_list.count === 0
+      end
+
+      def secondary_step_by_step
+        secondary_step_by_steps.first
```

Comment:
I'm not sure if this is needed as it's only being used in one place, to get the `current_step_nav`

Reply:
I've left it as this keeps the logic of choosing the secondary step by step away from the other functions

Reply:
I don't agree as the only place it's used, we're using `step_navs.first` in the condition above, but it's not a blocker.